src/MoBi.Core/Snapshots/Mappers/SimulationMapper.cs

```diff
@@ -66,6 +85,10 @@ public override async Task<MoBiSimulation> MapToModel(Simulation snapshot, Simul
 
       snapshot.OutputMappings?.Each(x => simulation.OutputMappings.Add(_outputMappingMapper.MapToModel(x, snapshotContextWithSimulation).Result));
 
+      updateParameters(simulation, snapshot.Parameters);
+
+      updateScaleDivisors(simulation, snapshot.ScaleDivisors);
```

Comment:
makes sense at the snapshot level to allow tracking and modifying scale factors, but at this time, they are not part of a configuration plan to swap values like parameters.